cmake/Utils.cmake

```diff
@@ -6,7 +6,7 @@ if(NOT CMAKE_PROPERTIES_TO_DUPLICATE)
   string(REGEX REPLACE ";" "\\\\;" CMAKE_PROPERTIES_TO_DUPLICATE "${CMAKE_PROPERTIES_TO_DUPLICATE}")
   string(REGEX REPLACE "\n" ";" CMAKE_PROPERTIES_TO_DUPLICATE "${CMAKE_PROPERTIES_TO_DUPLICATE}")
 
-  set(IGNORED_PROPERTIES "IMPORTED_GLOBAL" "NAME" "TYPE" "SOURCES" "HEADER_SETS" "INTERFACE_HEADER_SETS")
+  set(IGNORED_PROPERTIES "IMPORTED_GLOBAL" "NAME" "TYPE" "SOURCES" "HEADER_SETS" "INTERFACE_HEADER_SETS" "CXX_MODULE_HEADER_UNIT_SETS")
```

Comment:
I also had the issue with `CXX_MODULE_SETS`, `INTERFACE_CXX_MODULE_HEADER_UNIT_SETS` and `INTERFACE_CXX_MODULE_SETS`.

Reply:
I'll add those